scaleway/helpers_redis.go

```diff
@@ -106,3 +135,55 @@ func flattenRedisSettings(settings []*redis.ClusterSetting) interface{} {
 	}
 	return rawSettings
 }
+
+func flattenRedisPrivateNetwork(endpoints []*redis.Endpoint) (interface{}, bool) {
+	pnFlat := []map[string]interface{}(nil)
+	for _, endpoint := range endpoints {
+		if endpoint.PrivateNetwork != nil {
+			pn := endpoint.PrivateNetwork
+			pnZonedID := newZonedIDString(pn.Zone, pn.ID)
+			serviceIps := []interface{}(nil)
+			for _, ip := range pn.ServiceIPs {
+				serviceIps = append(serviceIps, ip.String())
+			}
+			pnFlat = append(pnFlat, map[string]interface{}{
+				"endpoint_id": endpoint.ID,
+				"zone":        pn.Zone,
+				"id":          pnZonedID,
+				"service_ips": serviceIps,
+			})
+		}
+	}
+	return pnFlat, len(pnFlat) != 0
+}
+
+func flattenRedisPublicNetwork(endpoints []*redis.Endpoint) interface{} {
+	pnFlat := []map[string]interface{}(nil)
+	for _, endpoint := range endpoints {
+		if endpoint.PublicNetwork != nil {
```

Comment:
Could you transform this like this:

```go
if endpoint.PrivateNetwork == nil {
    continue
}
...
```

by doing so, we are able to remove a level of indentation.